server/scripts/import-tests/gitOperations.js

```diff
@@ -3,7 +3,7 @@
 const spawn = require('cross-spawn');
 
 const ariaAtRepo = 'https://github.com/w3c/aria-at.git';
-const ariaAtDefaultBranch = 'master';
+const ariaAtDefaultBranch = 'move-resources'; // TODO: Revert to 'master' before merging
```

Comment:
Placing another reminder to change this before merging

Reply:
Also wondering that too. Will investigate

Reply:
It should be fixed now in https://github.com/w3c/aria-at-app/pull/1207/commits/0a545608a308a644447ff9b96c0fa7b9871c4455